features/api/services/settings.feature

```diff
@@ -44,7 +44,7 @@ Feature: Integration Settings
     Given I log in as provider "foo.3scale.localhost"
     And I go to the settings page for service "API" of provider "foo.3scale.localhost"
 
-    When I click on the label "APIcast self-managed"
+    When I click on the label "APIcast self-managed" and I confirm dialog box "This action will have the effect to clear out your Public Base URLs as well as to delete the corresponding routes in OpenShift"
```

Comment:
I think we're missing some tests for the opposite scenario `when I click on the label "..." and I cancel dialog box"..."` 🤔